consul/client.go

```diff
@@ -150,6 +151,10 @@ func (c *Client) setupSerf(conf *serf.Config, ch chan serf.Event, path string) (
 	conf.Tags["vsn_min"] = fmt.Sprintf("%d", ProtocolVersionMin)
 	conf.Tags["vsn_max"] = fmt.Sprintf("%d", ProtocolVersionMax)
 	conf.Tags["build"] = c.config.Build
+	conf.Tags["rpc_port"] = fmt.Sprintf("%d", c.config.RPCAddr.Port)
```

Comment:
We don't ever inbound connect to a client agent, right? We probably don't need this here.